exp-player/addon/components/exp-video-consent.js

```diff
@@ -0,0 +1,61 @@
+import Ember from 'ember';
+import ExpFrameBaseComponent from 'exp-player/components/exp-frame-base';
+import layout from '../templates/components/exp-video-consent';
+
+export default ExpFrameBaseComponent.extend({
+    layout,
+    videoRecorder: Em.inject.service(),
+
+    didInsertElement() {
+      this.get('videoRecorder').on('onUploadDone', () => {
+        this.get('videoRecorder').destroy();
+        this.get('videoRecorder').on('onUploadDone', null);
+        this.sendAction('next')
+      });
+      this.get('videoRecorder').start('Test', this.$('#recorder'), {record: false});
+    },
+
+    actions: {
+      record() {
+        this.get('videoRecorder').record();
+      },
+      finish() {
+        this.get('videoRecorder').stop();
+      }
+    },
+
+    meta: {
+        name: 'Video Consent Form',
+        description: 'A video consent form.',
+        parameters: {
+            type: 'object',
+            properties: {
+                id: {
+                    type: 'string',
+                    description: 'A unique identifier for this item'
+                },
+                title: {
+                    type: 'string',
+                    default: 'Notice of Consent'
+                },
+                prompt: {
+                    type: 'string',
+                    default: 'Say Words'
+                }
+            }
+        },
+        data: {
+            type: 'object',
+            properties: {
+                consentGranted: {
+                    type: 'boolean',
+                    default: false
+                },
+                videoId: {
```

Comment:
How does this information make it from the video recorder, to the component controller? I'm probably just missing something here.

(the items in `data` are what we expect to be sent to the server, though in the absence of validation this is more a notation convention than a rule)